### PR TITLE
Document describing steps to move an ALZ linked account "as is" to ASEA

### PR DESCRIPTION
**Pull request by ryanjwaters**
By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: docs/operations/operations-import-ALZAccount.md [hunk 1]
```diff
@@ -0,0 +1,206 @@
+# Operations - How to migrate an Amazon Landing Zone (ALZ) account "as is" into an Amazon Secure Environment Accelerator.
```
Comment (Brian969) [inline docs/operations/operations-import-ALZAccount.md:1]: Can you change Amazon to AWS in both spots?

File: docs/operations/operations-import-ALZAccount.md [hunk 2]
```diff
@@ -0,0 +1,206 @@
...
+
+<!-- TOC depthFrom:1 depthTo:4 -->
+
+- [1. Prerequisites (Setup)](#1-prerequisites-setup)
+- [2. Landing Zone - Disassociate the account from the Landing Zone](#2-landing-zone---disassociate-the-account-from-the-landing-zone)
+- [3. Landing Zone - Remove the account from the landing Zone (ALZ) organizations and make standalone](#3-landing-zone---remove-the-account-from-the-landing-zone-alz-organizations-and-make-standalone)
+- [4. Accelerator (ASEA) - Invite the account into it's organization](#4-acceleartor-asea---invite-the-account-into-its-organization)
```
Comment (Brian969) [inline docs/operations/operations-import-ALZAccount.md:8]: it's to its

Comment (Brian969) [reply docs/operations/operations-import-ALZAccount.md:8]: I realize markdown is tough, but could you cut-paste this into Word - run through spell and grammar checker and migrate changes back here?  (would make it quicker to review/cleanup)

File: docs/operations/operations-import-ALZAccount.md [hunk 3]
```diff
@@ -0,0 +1,206 @@
...
+
+### 1.1.
+
+If working with your AWS account team (TAM/SA), they will reach out to an internal team to request the linked account be switched to invoicing.  This way the customer doesn’t have to enter a credit card when making the account standalone in the upcoming steps.
+
+### 1.2.
+
+Confirm access to the email account associated to the ALZ linked account being migrated to the ASEA, as well as access to the AWS linked account itself.  This runbook will first make the account standalone (remove from ALZ organizations) so one needs to make sure the customer has root access to the account and that the email address attached to the account is available before running this procedure.  If needed, you can reset the linked account root password by following: https://docs.aws.amazon.com/IAM/latest/UserGuide/id_credentials_passwords_change-root.html
```
Comment (Brian969) [inline docs/operations/operations-import-ALZAccount.md:29]: ", as well as access to the AWS linked account itself" - what does this mean.  also grammar/the way tacked onto sentence is awkward.

File: docs/operations/operations-import-ALZAccount.md (hunk 1 above)
Comment (ryanjwaters) [reply docs/operations/operations-import-ALZAccount.md]: Done - made the changes  and had one commit with spelling, grammar, added some more subtitles and tried to simplify where possible.   Please feel free to edit changes directly.

File: docs/operations/operations-import-ALZAccount.md (hunk 2 above)
Comment (ryanjwaters) [reply docs/operations/operations-import-ALZAccount.md]: Done - made the changes  and had one commit with spelling, grammar, added some more subtitles and tried to simplify where possible.   Please feel free to edit changes directly.

File: docs/operations/operations-import-ALZAccount.md (hunk 3 above)
Comment (ryanjwaters) [reply docs/operations/operations-import-ALZAccount.md]: Done - made the changes  and had one commit with spelling, grammar, added some more subtitles and tried to simplify where possible.   Please feel free to edit changes directly.

**Review by Brian969 — APPROVED** [review]
LGTM

**Pull request merged**